test/TestAssets/TestProjects/DotnetRunTargetExtensionWithError/DotnetRunTargetExtensionWithError.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <Target Name="TriggerErrorDuringSetRunArgs" BeforeTargets="ComputeRunArguments">
+    <Warning Code="MYAPP001" Text="This app is wonky, dude, don't run it" />
+    <Error Code="MYAPP002" Text="I _said_, don't run it!" />
+  </Target>
```

Comment:
@fabiocav and here we're showing that if a target issues diagnostics that we report them to the user (so we fail the run if there are errors while determining the run arguments). This means that you can enforce invariants you need in your targets.